eZ/Publish/API/Repository/Values/URL/Query/Criterion/Validity.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace eZ\Publish\API\Repository\Values\URL\Query\Criterion;
+
+/**
+ * Matches URLs based on validity flag.
+ */
+class Validity extends Matcher
+{
+    /**
+     * @var bool|null
+     */
+    public $isValid;
+
+    /**
+     * Validity constructor.
+     *
+     * @param bool|null $isValid
```

Comment:
is there any point in building a isValid criterion with null?

Reply:
No, this field cannot be null. 7db2e3abdb0ef8050d2090630ebd26c5a5656486